scalalib/test/src/HelloWorldTests.scala

```diff
@@ -179,6 +179,7 @@ object HelloWorldTests extends TestSuite {
         developers =
           Seq(Developer("lihaoyi", "Li Haoyi", "https://github.com/lihaoyi"))
       )
+      override def versionScheme = Some(VersionScheme.EarlySemVer)
```

Comment:
Can you please add the link to the documentation of the version scheme concept. Either here or at `VersionScheme`.

Reply:
And a proper return type, please.

Reply:
We should no longer add targets without Scaladoc (including a `@since`).

Reply:
This is the test. The definition has already scaladoc. I have to add the `@since` tho.

Reply:
I realize that the documentation about early-semver is a bit spread over multiple places. Relevant information is here:
https://docs.scala-lang.org/overviews/core/binary-compatibility-for-library-authors.html#recommended-versioning-scheme
https://www.scala-lang.org/blog/2021/02/16/preventing-version-conflicts-with-versionscheme.html
https://www.scala-sbt.org/1.x/docs/Publishing.html#Version+scheme

Reply:
Sorry for the wrong place.